test/src/utils/transition-manager.spec.js

```diff
@@ -404,6 +406,9 @@ test('TransitionManager#TRANSITION_EVENTS', t => {
         testCase.shouldChange[mode].transitionEasing,
         'transitionEasing match'
       );
+
+      // Clean up
+      transitionManager._endTransition();
```

Comment:
Why this is needed to be called in tests? Is this a step that whoever uses TransitionManager's also need to do?

Reply:
We provided an external timer so the animation never ends. The test cannot end if there is a pending animation frame requested.